pom.xml

```diff
@@ -74,6 +74,27 @@
       <groupId>io.quarkus</groupId>
       <artifactId>quarkus-smallrye-health</artifactId>
     </dependency>
+    <dependency>
+      <groupId>io.quarkus</groupId>
+      <artifactId>quarkus-smallrye-openapi</artifactId>
+    </dependency>
+    <dependency>
+      <groupId>io.quarkus</groupId>
+      <artifactId>quarkus-smallrye-fault-tolerance</artifactId>
+      <version>3.6.1</version>
```

Comment:
Is this version needed? Does not come from managedDependency (quarkus-bom)?

Reply:
deleted